include/swift/AST/Decl.h

```diff
@@ -3500,12 +3500,17 @@ class NominalTypeDecl : public GenericTypeDecl, public IterableDeclContext {
   /// Find, or potentially synthesize, the implicit 'id' property of this actor.
   VarDecl *getDistributedActorIDProperty() const;
 
-  /// Find the 'RemoteCallTarget.init(_:)' initializer function
+  /// Find the 'RemoteCallTarget.init(_:)' initializer function.
   ConstructorDecl* getDistributedRemoteCallTargetInitFunction() const;
 
-  /// Find the 'RemoteCallArgument(label:name:value:)' initializer function
+  /// Find the 'RemoteCallArgument(label:name:value:)' initializer function.
   ConstructorDecl* getDistributedRemoteCallArgumentInitFunction() const;
 
+  /// Find the
+  /// 'DistributedActorSystem.invokeHandlerOnReturn(handler:value:metatype:)
+  /// function.
+  FuncDecl *getDistributedActorSystemInvokeHandlerOnReturnFunction() const;
```

Comment:
Is this a leftover code that has to be deleted? I don't see any usages of it and request associated with this method returns `nullptr` always with a `// FIXME`.

Reply:
Ouch thanks for noticing; yes this is leftover indeed.